gems/attachment_fu/lib/attachment_fu/railtie.rb

```diff
@@ -25,7 +25,7 @@ class Railtie < ::Rails::Railtie
     end
 
     def self.setup_tempfile_path
-      AttachmentFu.tempfile_path = "/tmp/attachment_fu"
+      AttachmentFu.tempfile_path = "tmp/attachment_fu"
```

Comment:
We would prefer not to change the default here, but making this configurable would seem reasonable.  Currently it looks like you can set `ATTACHMENT_FU_TEMPFILE_PATH` in initialization code; maybe making it also support an environment variable or reading it from a YML file would be reasonable.

Reply:
project uses dir: canvasdir/tmp for all temporary and uploaded files (if no s3), but uses system temp for uploaded temp files.

Sorry, I'm admin, not ruby developer. So I can not do more seriuous code changes.
Otherwise I whould offer redis gem update. But when I try newer version, some more code needs to be changed.